tests/test_parseutils.py

```diff
@@ -1,3 +1,5 @@
+# mypy: ignore-errors
```

Comment:
Is there a way to ignore all the errors in the directory rather than at each file in test directory?